### ISSUE TITLE
Use Site Diagnostics are missed in tuple symbols in VB

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#61112]
https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleEventSymbol.vb#L87
https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleFieldSymbol.vb#L102
https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleMethodSymbol.vb#L133
https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/Symbols/Tuples/TuplePropertySymbol.vb#L115

All of these call `MergeUseSiteInfo` and discard the result. In the VB compiler, that method does not take use site info by ref, it returns a new copy, so all of these locations will miss any use site info that should be reported.

**Comment by AlekseyTs** [comment]
@333fred It looks like ```MergeUseSiteInfo``` was changed to take ```UseSiteInfo``` by ref and return a bool. Does this resolve the issue?

**Comment by 333fred** [comment]
I left this issue open for adding test coverage. The bug itself was fixed, but we have no validation of the missing diagnostics.
